client-wasm/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2024-07-01"
```

Comment:
I'm not sure we actually need to use nightly anymore; I only see 2 features enabled and they're in the node (and can actually be removed).

Reply:
Nvm, I grepped in the wrong place; there's a bunch of features we don't need but there's 2 that we do need so we need to keep relying on nightly for now.